cep-XXXX.md

```diff
@@ -0,0 +1,424 @@
+The `repodata_from_packages.json`, `patch_instructions.json`, and `run_exports.json` artifacts MAY not be present
+on all OCI conda channels.
+
+The OCI tag `latest` MUST always refer to the copy of the repodata / run exports currently in use.
+
+Older copies of the repodata / run exports MAY be stored as well. If older copies of the repodata / run exports are stored,
+they MUST use an OCI tag that is the UTC time formatted as `YYYYMMDDThhmmssZ`.
+
+### Channeldata Artifact Names and Tags
+
+The `channeldata.json` for an OCI conda channel MUST be stored as an OCI artifact as
+
+```text
+<OCI conda channel base URL>/channeldata.json:latest
+```
+
+where the contents and mediaTypes of the OCI blobs that make up the `channeldata.json` artifact are defined below.
+
+The OCI tag `latest` MUST always refer to the copy of the `channeldata.json` currently in use.
+
+Older copies of the `channeldata.json` MAY be stored as well. If older copies of the `channeldata.json` are stored, they
+MUST use an OCI tag that is the UTC time formatted as `YYYYMMDDThhmmssZ`.
+
+### Further Stipulations Related to OCI conda Chennal Repositories and Tags
+
+- If either of the `<OCI-encoded package name>` or the OCI tag for the conda package are hashed, then both MUST be hashed.
+- If an OCI registry implementation errors due to length when dealing with the full OCI conda URL for the package, then
+  that package cannot be stored in that registry and an error MUST be raised.
+- The encoding rules defined above are solely for internal storage of conda packages within an OCI registry. Outside of
+  dealing with the OCI registry itself, the original forms of the channel, subdir, label, name, version and build string MUST be used.
+
+### Allowed Blob mediaTypes
+
+We define the following custom media types that MUST be used for the storage of conda repodata, run exports, channel data,
+and packages in an OCI registry:
+
+| Blob type            | Content type                | mediaType                                      |
+|----------------------|-----------------------------|------------------------------------------------|
+| conda package v1     | .tar.bz2 package            | application/vnd.conda.package.v1               |
+| conda package v2     | .conda package              | application/vnd.conda.package.v2               |
+| package info         | `info` folder as gzip       | application/vnd.conda.info.v1.tar+gzip         |
+| package index        | `info/index.json` file      | application/vnd.conda.info.index.v1+json       |
+| repodata.json        | `repodata.json` file        | application/vnd.conda.repodata.v1+json         |
+| repodata.json.zst    | `repodata.json.zst` file    | application/vnd.conda.repodata.v1+json+zst     |
+| repodata.json.gz     | `repodata.json.gz` file     | application/vnd.conda.repodata.v1+json+gzip    |
+| repodata.json.bz2    | `repodata.json.bz2` file    | application/vnd.conda.repodata.v1+json+bz2     |
```

Comment:
Do we need to add `v2` repodatas (CEP 15)?

Reply:
Yes